Plugins/GitUIPluginInterfaces/IGitUICommands.cs

```diff
@@ -30,9 +30,8 @@ public interface IGitUICommands
         bool StartCommandLineProcessDialog(IWin32Window owner, IGitCommand command);
         void StartBatchFileProcessDialog(string batchFile);
 
-        bool StartRemotesDialog();
```

Comment:
Since StartRemotesDialog was removed, is it possible to add
`
 bool RunCommand(IReadOnlyList<string> args)
`
from GitUICommands or is there an other way to use this now?

My request comes from the gerrit plugin perspective

Reply:
Sorry, @thimmy687, I was not aware that the interface `IGitUICommands.StartRemotesDialog()` was / is in use.
The `FormRemotes` cannot be opened by means of the CLI. So I think `RunCommand(IReadOnlyList<string> args)` would not work for your usecase.
Would it suffice to restore `IGitUICommands.StartRemotesDialog()`?

Reply:
I have a workaround for it in this PR https://github.com/gitextensions/GitExtensions.GerritPlugin/pull/16 

So we could keep it how it is